packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -0,0 +1,386 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:file/file.dart';
+import 'package:flutter_tools/src/base/logger.dart';
+import 'package:flutter_tools/src/bundle.dart';
+
+import '../aot.dart';
+import '../application_package.dart';
+import '../artifacts.dart';
+import '../base/common.dart';
+import '../base/file_system.dart';
+import '../base/process.dart';
+import '../base/utils.dart';
+import '../build_info.dart';
+import '../build_system/targets/ios.dart';
+import '../globals.dart';
+import '../macos/cocoapod_utils.dart';
+import '../macos/xcode.dart';
+import '../plugins.dart';
+import '../project.dart';
+import '../runner/flutter_command.dart' show DevelopmentArtifact, FlutterCommandResult;
+import 'build.dart';
+
+class BuildIOSFrameworkCommand extends BuildSubCommand {
```

Comment:
Here and elsewhere, it would nice to have doc comments

Reply:
I noticed most of the files in commands/ don't have any. Added here an in various adjacent files.

Reply:
Thank you!